sys/linux/futex.txt

```diff
@@ -4,6 +4,7 @@
 include <linux/futex.h>
 
 futex(addr ptr[in, int32[0:2]], op flags[futex_op], val int32[0:2], timeout ptr[in, timespec], addr2 ptr[in, int32[0:2]], val3 int32[0:2])
+futex_waitv(addr ptr[in, array[futex_wait_block, 0:127]], val len[addr], flg const[0], timeout ptr[in, timespec], val2 flags[futex_clock_id])
```

Comment:
Maybe we can use `clockid` instead of `val2`?

Reply:
Sure.